### DISCUSSION TITLE
Game Compatibility Information Collection

### DISCUSSION THREAD

**Discussion by SheffeyG** [SheffeyG/CheatDeck#23] [Show and tell]
This post is for collecting games and cheats compatibility information, as most cheats and trainers are not designed for SteamOS, some of them could work perfectly while some are not, you can share the information about the game and cheat that works perfectly with cheatdeck here.
For example:
```
Game: Hollow Knight v1.5.68
Trainer: Hollow.Knight.v1.5.68.Plus.11.Trainer-FLiNG Build.2021.06.13 from flingtrainer.com
Compatibility: GE-Proton8-9
```

**Comment by GrizzlyFoxCat** [comment]
NOT WORKING
Hollow Knight v1.5.78
Trainer: Hollow.Knight.v1.5.68-v1.5.80.Plus.11.Trainer-FLiNG
Compatibility: Proton experimental

**Comment by Nyn69** [comment]
NOT WORKING
Rune Factory 3 Special v1.0.4
Trainer: Rune.Factory.3.Special.v1.0.Plus.22.Trainer-FLiNG
Compatibility: GE-Proton9-23 / Proton 9.0-4 / Experimental

**Comment by Darknessfell** [comment]
Second window doesn't show up
Game: Kingdom Hearts -HD 1.5+2.5 ReMIX-
trainer: Kingdom Hearts Final Mix v1.0 Plus 22 Trainer.exe
Compatibility: Unchecked from game's settings
